### PR TITLE
Changes to index-clip.md

### PR DESCRIPTION
**Pull request by Florian-H-Lab**

### CODE REVIEW

File: index-clip.md [hunk 1]
```diff
@@ -51,11 +53,28 @@ You can choose between three different types of peak calling for the data analys
 
 **Table 1**: Data specification of the different peak calling algorithms.
 
-| Tool | Biological Replicates (Yes/No) | Control Data (Yes/No) |
-| ---            | :-:      | :-:     |
-| <a href="https://github.com/tbischler/PEAKachu">PEAKachu</a>            | Yes | Yes   |
-| <a href="https://doi.org/10.1186/s13059-017-1364-2">PureCLIP</a>            | No | Yes   |
-| <a href="https://doi.org/10.1093/bioinformatics/bts569">Piranha</a>           | No | No   |
+<table class="table table-striped">
```
Comment (erasche) [inline index-clip.md]: If you just wanted to make the old table striped you can just include

```
[table stuff]
{: .table.table-striped}
```

File: index-clip.md [hunk 2]
```diff
@@ -33,12 +33,14 @@ Galaxy CLIP-Explorer can process large CLIP-Seq data of eCLIP and iCLIP. We proc
 
 We provide the subsequent workflows to automatize the data analysis for iCLIP and eCLIP data. All workflows can be found [here](https://github.com/Florian-H-Lab/CLIP-Explorer). The data needs to be in FASTA or FASTQ format and can be either multiplexed or de-multiplexed. All workflows, except the robust peak analysis, require the data as a list of dataset pairs. A tutorial to create a list of dataset pairs can be found in the CLIP-Seq data analysis <a target="_parent" href="https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/clipseq/tutorial.html">tutorial</a> or [here](https://galaxyproject.github.io/training-material/topics/galaxy-data-manipulation/tutorials/collections/tutorial.html). Please have in mind that all workflows need additional input files from the user.
 
-### 3.1 From scratch for de-multiplexing FASTQ files
+### 3.1 From scratch to de-multiplex FASTQ files
```
Comment (erasche) [inline index-clip.md]:
```suggestion
### 3.1 From scratch to de-multiplexed FASTQ files
```

File: index-clip.md (hunk 1 above)
Comment (Florian-H-Lab) [reply index-clip.md]: Oh well xD didn't know about the easier way. I looked into the code of the galaxy webpages which lead me to my solution ^^

**Comment by bgruening** [comment]
ping @Florian-H-Lab

**Pull request merged**